packages/lib/turndownService.ts

```diff
@@ -2,11 +2,48 @@ import TurndownService from "turndown";
 
 const turndownService = new TurndownService();
 
-turndownService.addRule("newLine", {
-  filter: ["br"],
-  replacement: () => {
+function turndown(html: string | TurndownService.Node): string {
+  let result = turndownService.turndown(html);
+  result = result.replaceAll("[<p><br></p>]", "");
```

Comment:
This is to avoid an error in the rare case someone tries to set a link in an empty line